tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -11,6 +11,7 @@ Given('an intercepted search response', () => {
 });
 
 Then('start button is clicked', () => {
+  cy.get('div.x').should('exist');
```

Comment:
We try to avoid using classes in testing selectors, and we obviously completely forbid tag selectors 🤣. Add a data-test selector to the `.x` div and use it here:

```suggestion
  cy.getByDataTest('x').should('exist');
```